frontend/express/public/javascripts/pre-login.js

```diff
@@ -49,6 +49,48 @@ function encodeSomeHtml(html, options) {
     }
 }
 
+
+/**
+* Check password Validation.
+* @param {string} password - password string
+* @return {bool} result - return password is validated or not.
+* @memberof Pre Login
+* 
+/* exported passwordCheck */
+function passwordCheck(password) {
```

Comment:
Hello,
I think in ticket I mentioned to do server side validation.
Mostly because it must work based on config values, not on hardcoded values.

Reply:
OK updated.